lib/ansible/modules/monitoring/zabbix_host.py

```diff
@@ -62,7 +63,7 @@
         description:
             - Visible name of the host in Zabbix.
         required: false
-        version_added: '2.3'
+        version_added: '2.4'
```

Comment:
was this changed by mistake?

Reply:
The original PR was created beginning of Januari, at that time it was `version_added: 2.3`. A lot of time was passed, git troubles because of that and thinking if I close the old pr and create a new one, maybe this PR will be accepted/merged sooner ....

Reply:
we probably have a misunderstanding... this `version_added` is related to `visible_name` and usually does not need to be updated.

Reply:
ah, maybe I updated a `version_added` to much. Will set this back to 2.3.